health/health_server.go

```diff
@@ -18,8 +18,9 @@
 
 //go:generate ./regenerate.sh
 
-// Package health provides some utility functions to health-check a server. The implementation
-// is based on protobuf. Users need to write their own implementations if other IDLs are used.
+// Package health enables support for gRPC health checking. It is EXPERIMENTAL.
```

Comment:
There are two parts to this package.  A service implementation and the client-side health checks.  The _service_ can be implemented with a different proto library (e.g. gogoproto) but not a different IDL.  The client can't be implemented by the user because it needs internal.  I'd leave that stuff out entirely, personally.

But please mention that there's a service here that users can use to expose server health.  And mention that this package must be imported to enable support for client-side health checks, if configured in the service config.